libs/scripts/preview/preview.js

```diff
@@ -0,0 +1,296 @@
+/*
+ * Copyright 2022 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+/* eslint-disable no-console */
+
+import {
+  toClassName,
+  getMetadata,
+  getConfigForFullExperiment,
+  getConfigForInstantExperiment,
+  // checkTesting,
+} from '../personalization.js';
+
+const checkTesting = () => true;
+const getExperimentConfig = (experiment, instantExperiment) => {
+  if (instantExperiment) {
+    return getConfigForInstantExperiment(experiment, instantExperiment);
+  }
+  return getConfigForFullExperiment(experiment);
```

Comment:
not sure how `getConfigForFullExperiment` works, but we should be able to do `getConfigForFullExperiment(experiment, instantExperiment)` and check whether `instantExperiment` is defined inside the method